### PR TITLE
Use the built in .net compression capabilities

### PR DESCRIPTION
**Pull request by mcnallys**
Removing the need to have sharpziplib embedded into the library. Reduces assembly size and provides slightly better compression now.

### CODE REVIEW

File: src/PdfSharper/Pdf.Filters/FlateDecode.cs [hunk 1]
```diff
@@ -59,162 +54,47 @@ public override byte[] Encode(byte[] data)
...
-            //
-            //       0 - compressor used fastest algorithm
-            //       1 - compressor used fast algorithm
-            //       2 - compressor used default algorithm
-            //       3 - compressor used maximum compression, slowest algorithm
-            //
-            //    The information in FLEVEL is not needed for decompression; it
-            //    is there to indicate if recompression might be worthwhile.
-            ms.WriteByte(0x49);
+                DeflateStream zip = new DeflateStream(ms, CompressionMode.Compress, true);
```
Comment (pex-simsr) [inline src/PdfSharper/Pdf.Filters/FlateDecode.cs]: Wrap DeflateStream in Using..

File: src/PdfSharper/Pdf.Filters/FlateDecode.cs [hunk 2]
```diff
@@ -59,162 +54,47 @@ public override byte[] Encode(byte[] data)
...
-            msOutput.Flush();
-            if (msOutput.Length >= 0)
-            {
-#if NETFX_CORE || UWP
-                return msOutput.ToArray();
-#else
-                msOutput.Capacity = (int)msOutput.Length;
-                return msOutput.GetBuffer();
-#endif
+                DeflateStream zip = new DeflateStream(msInput, CompressionMode.Decompress, true);
```
Comment (pex-simsr) [inline src/PdfSharper/Pdf.Filters/FlateDecode.cs]: Wrap DeflateStream in Using

**Review by pex-simsr — CHANGES_REQUESTED** [review]
DeflateStream object should be wrapped in a Using.. looks good.

File: src/PdfSharper/Pdf.Filters/FlateDecode.cs (hunk 2 above)
Comment (mcnallys) [reply src/PdfSharper/Pdf.Filters/FlateDecode.cs]: Good catch.

File: src/PdfSharper/Pdf.Filters/FlateDecode.cs (hunk 1 above)
Comment (mcnallys) [reply src/PdfSharper/Pdf.Filters/FlateDecode.cs]: Good catch.

**Review by pex-simsr — APPROVED** [review]

**Pull request merged**